e2e/errors.go

```diff
@@ -23,6 +23,8 @@ import (
 	utilnet "k8s.io/apimachinery/pkg/util/net"
 )
 
+const errRWOPConflict = "node has pod using PersistentVolumeClaim with the same name and ReadWriteOncePod access mode."
```

Comment:
This is a string, not an Error. Why is it in this file if it is never used here?

Reply:
sure, moved to proper place.